### PR TITLE
feat: add support for Nutanix builds

### PR DESCRIPTION
**Pull request by mjnagel**
Closes https://github.com/defenseunicorns/uds-rke2-image-builder/issues/8

### CODE REVIEW

File: packer/aws/variables.pkr.hcl [hunk 1]
```diff
@@ -17,7 +17,7 @@ variable "base_ami_name" {
 variable "rke2_version" {
   type        = string
   description = "RKE2 version to install on the AMI"
-  default     = "v1.26.7+rke2r1"
+  default     = "v1.27.5+rke2r1"
```
Comment (mjnagel) [inline packer/aws/variables.pkr.hcl:20]:
```suggestion
  default     = "v1.26.7+rke2r1"
```
Forgot I changed this, going to check if this was the issue with CI...if so will open a ticket to track updating to 1.27.x

File: Makefile [hunk 2]
```diff
@@ -53,15 +56,37 @@ validate-ami-rhel: ## Run packer validation for the AWS RHEL AMI.
...
 # Test Targets
 ######################
 
 .PHONY: test-ami-ubuntu
-test-ami-ubuntu: fmt-ami validate-ami-ubuntu build-ami-ubuntu ## fmt, validate, and build the AMI for AWS.
+test-ami-ubuntu: fmt-ami validate-ami-ubuntu build-ami-ubuntu ## fmt, validate, and build the Ubuntu AMI for AWS.
 
 .PHONY: test-ami-rhel
-test-ami-rhel: fmt-ami validate-ami-rhel build-ami-rhel ## fmt, validate, and build the AMI for AWS.
+test-ami-rhel: fmt-ami validate-ami-rhel build-ami-rhel ## fmt, validate, and build the Ubuntu AMI for AWS.
```
Comment (jacobbmay) [inline Makefile:89]:
```suggestion
test-ami-rhel: fmt-ami validate-ami-rhel build-ami-rhel ## fmt, validate, and build the RHEL AMI for AWS.
```

**Review by jacobbmay — APPROVED** [review]
LGTM

**Pull request merged**